From 79ccfa90a141827f9911fa8e86dc7eb224a305be Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Mon, 10 Jul 2023 22:01:16 -0500
Subject: [PATCH 01/57] tests/util/grub-shell: Add --verbose to grub-mkrescue
 when $debug is greater than 2

Since this is fairly verbose output, do not enable first level of debug
is turned on.

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
---
 tests/util/grub-shell.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/util/grub-shell.in b/tests/util/grub-shell.in
index a0ca18e41..5cbe79447 100644
--- a/tests/util/grub-shell.in
+++ b/tests/util/grub-shell.in
@@ -422,7 +422,9 @@ fi
 
 if [ x$boot != xnet ] && [ x$boot != xemu ]; then
     pkgdatadir="${builddir}" \
-    exec_show_error "${builddir}/grub-mkrescue" "--output=${isofile}" \
+    exec_show_error "${builddir}/grub-mkrescue" \
+	${debug:+$([ "$debug" -gt 2 ] && echo -n "--verbose")} \
+	"--output=${isofile}" \
 	"--override-directory=${builddir}/grub-core" \
 	--rom-directory="${rom_directory}" \
 	--locale-directory="${srcdir}/po" \
-- 
2.42.0

